colorizer_data/converter.py

```diff
@@ -138,12 +147,24 @@ def _write_data(
     writer: ColorizerDatasetWriter,
     config: ConverterConfig,
 ):
+    outliers_data = _get_data_or_none(dataset, config["outlier_column"])
```

Comment:
same comment I had on previous PR, I'm not sure if dev tooling knows what the fields of config are when they are accessed by string, and my impression is that `config.outlier_column` is a less error prone syntax for this kind of data structure (like DataClass maybe?).  But if things are actually simpler this way, then so be it

Reply:
For a DataClass it makes sense! Right now this is using a TypedDict instead, which I think is standard to use a string accessor? (or at least VSCode Intellisense seems to work with string accessors best)...

Hm. Maybe I *should* change this to a DataClass so I can attach comments.